gkr_iop/src/gkr/mock.rs

```diff
@@ -63,7 +63,7 @@ impl<E: ExtensionField> MockProver<E> {
         let field_type_default = FieldType::default();
         let mut evaluations = evaluations; // Change evaluations' lifetime.
         evaluations.resize_with(circuit.n_evaluations, || &field_type_default);
-        challenges.resize_with(circuit.n_challenges, || E::random(&mut rng));
+        challenges.resize_with(2 + circuit.n_challenges, || E::random(&mut rng));
```

Comment:
Why is it `+2`?

Reply:
index 0, 1 is for alpha, beta global challenge. Here `circuit.n_challenges` only counting local challenges, which is 0.
we can make this converation open without Resolve 👍 , so it can be refer from future task